_docs/_latest/concepts/best-practices.md

```diff
@@ -113,4 +272,126 @@ whitelist IP addresses.
+monitoring tools. Best practice is to [use Stackdriver Monitoring to monitor
+your resources and provide
+alerts](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#use_stackdriver_monitoring_to_monitor_your_resources_and_provide_alerts).
+
+**Logging Exports**
+
+[Export logs to BigQuery for analysis and long term
+storage](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#export_logs_to_bigquery_for_analysis_and_long_term_storage).
+
+**Cloud Audit Logs**
+
+[Admin actions, such as creating, updating, and deleting a resource are
+captured in the Cloud Audit
+Logs](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#monitor_administrative_actions).
+Requirements should be determined on retaining or alerting on these actions.
+
+
+**Data Access Logs**
+
+For finer grained access logs, [Data Access logs can be
+enabled](https://cloud.google.com/logging/docs/audit/configure-data-access) to
+capture read operations being performed by users in the project.
+
+**Aggregated Logging Exports**
+
+An organization can create an [aggregated
+export](https://cloud.google.com/logging/docs/export/aggregated_exports) sink
+that can export log entries from all the projects, folders, and billing
+accounts of the organization. As an example, an organization might use this
+feature to export audit log entries from its projects to a central location.
+
+## Cost Control
+
+**Billing Alerts**
+
+[Billing
+alerts](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#set_alerts_for_monthly_spending_thresholds)
+can be configured for a project or a billing account for monthly spending
+thresholds. There is a limitation that these alerts will only go to the Billing
+Admin.
+
+
+**Labels**
+
+For finer grained cost attribution across resources within or across projects
+project
+[labels](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#use_project_labels_to_further_categorize_projects_in_billing_export)
+can be used to further categorize projects in the billing export.
+
+
+**Billing Admins**
+
+Organizations should review their current Billing Access Control following the
+principle of least privilege.
+Check [how to bill](https://cloud.google.com/billing/docs/how-to/billing-access).
+
+**Invoiced Billing Account**
+
+Most large enterprise accounts should be using an offline billing account.
+
+**Whitelisting the Customer Billing Account**
+
+For large customers not on invoiced billing, their billing account should be
+whitelisted by Google. This will prevent their account from being suspended or
+cancelled automatically - instead Google support will reach out to the Google
+account team and the customer before taking action.
+
+**Cost Attribution with Projects**
+
+Projects are a good standard way of attributing costs. If the naming convention
```

Comment:
Address the customer directly:

Projects are a good, standard way to attribute costs. If you use the recommended [company tag]-[group tag]-[system name]-[env] convention, you can attribute costs to a Business Unit > System > Environment level of granularity. The monthly bill separates costs by project, then by resource type and labels. To learn more, see the [enterprise guide](https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#your_monthly_bill_breaks_down_costs_by_project_then_by_resource_type).